nvdaHelper/vbufBase/storage.cpp

```diff
@@ -1159,6 +1176,28 @@ bool VBufStorage_buffer_t::isNodeInBuffer(VBufStorage_fieldNode_t* node) {
 	return this->nodes.count(node)?true:false;
 }
 
+bool VBufStorage_buffer_t::moveSubtree(VBufStorage_fieldNode_t* node, VBufStorage_controlFieldNode_t* newParent, VBufStorage_fieldNode_t* newPrevious) {

```

Comment:
This method isn't used by these changes and should be removed (or used in replaceSubtrees).